services-operator/test/test_beamsqlstatementsetoperator.py

```diff
@@ -0,0 +1,308 @@
+from unittest import TestCase
+import unittest
+import shlex
+import subprocess
+import time
+from bunch import Bunch
+from mock import patch, MagicMock
+import kopf
+from kopf.testing import KopfRunner
+
+
+import beamsqlstatementsetoperator as target
+
+
+"""
+Mock functions
+"""
+class Logger():
+            def info(self, message):
+                pass
+            def debug(self, message):
+                pass
+            def error(self, message):
+                pass
+            def warn(self, message):
+                pass
+
+def kopf_info(body, reason, message):
+    pass
+
+class TestInit(TestCase):
+    @patch('kopf.info', kopf_info)
+    def test_init(self):
+        body = {
+            "metadata": {
+                "name": "name",
+                "namespace": "namespace"
+            },
+            "spec": {}
+        }
+        patch = Bunch()
+        patch.status = {}
+        result = target.create(body, body["spec"], patch, Logger())
+        assert result['createdOn'] is not None
+        assert patch.status['state'] == "INITIALIZED"
```

Comment:
This would result in a crash if the test fails (other tests would still run). We actually want to differentiate between crashes and fails, so better use `self.assert_equal` instead of `assert a == b`